config/activity_labels/recipe_dessertquesadilla.yaml

```diff
@@ -0,0 +1,50 @@
+version: "1"
+title: "Dessert Quesadilla"
+labels:
+  # Item:
+  # - id: Integer identification number of the label.
+  # - label: Short human-readable semantic string to succinctly represent the
+  #          label.
+  - id: 0
+    label: "background"
+  - id: 1
+    label: "place-tortilla"
+    full_str: "Place tortilla on cutting board"
+    depends: ["start"]
+  - id: 2
+    label: "scoop-nutella"
+    full_str: "Use the butter knife to scoop nutella from the jar"
+    depends: ["start"]
+  - id: 3
+    label: "spread-nutella-tortilla"
+    full_str: "Spread nutella onto tortilla, leaving ½ inch uncovered at the edges"
+    depends: ["place-tortilla", "scoop-nutella"]
+  - id: 4
+    label: "slice-banana"
+    full_str: "Slice banana"
+    depends: ["start"]
+  - id: 5
+    label: "top-banana-slices"
+    full_str: "Top with banana slices."
+    depends: ["slice-banana", "spread-nutella-tortilla"]
+  - id: 6
+    label: "sprinkle-cinnamon"
+    full_str: "Sprinkle small amount of cinnamon onto tortilla"
+    depends: ["spread-nutella-tortilla"]
+  - id: 7
+    label: "fold-tortilla-half"
+    full_str: "Fold tortilla in half into semi-cirlce"
+    depends: ["sprinkle-cinnamon", "top-banana-slices"]
+  - id: 8
+    label: "slice-tortilla-half"
+    full_str: "Slice tortilla in half using butter knife to create two triangular wedges"
+    depends: ["fold-tortilla-half"]
+  - id: 9
+    label: "place-tortilla-wedge"
+    full_str: "Place tortilla wedge on the plate"
+    depends: ["slice-tortilla-half"]
+    repeat: 2
+  - id: 10
+    label: "done"
+    full_str: "done"
+    depends: ["place-tortilla-wedge-all"]
```

Comment:
What is this label referring to? Are there other "dangling" dependency references? This is where a validation script/schema would be helpful.